mathesar_ui/src/pages/database-connection/AddDatabaseConnection.svelte

```diff
@@ -0,0 +1,43 @@
+<script lang="ts">
+  import AppendBreadcrumb from '@mathesar/components/breadcrumb/AppendBreadcrumb.svelte';
+  import { iconAddNew } from '@mathesar/icons';
+  import {
+    DATABASE_CONNECTION_ADD_URL,
+    getDatabasePageUrl,
+  } from '@mathesar/routes/urls';
+  import { toast } from '@mathesar/stores/toast';
+  import type { Database } from '@mathesar/AppTypes';
+  import { router } from 'tinro';
+  import { reloadDatabases } from '@mathesar/stores/databases';
+  import { reflectApi } from '@mathesar/api/reflect';
+  import FormBox from '../admin-users/FormBox.svelte';
```

Comment:
Can we move this FormBox to a more common location. It currently seems to be specific to a different route, and referencing components from across routes reduces modularity.